packages/contracts-bedrock/deploy/017-SystemConfigImpl.ts

```diff
@@ -18,6 +23,14 @@ const deployFn: DeployFunction = async (hre) => {
       batcherHash,
       hre.deployConfig.l2GenesisBlockGasLimit,
       hre.deployConfig.p2pSequencerAddress,
+      {
+        maxResourceLimit: 20_000_000,
+        elasticityMultiplier: 10,
+        baseFeeMaxChangeDenominator: 8,
+        systemTxMaxGas: 1_000_000,
+        minimumBaseFee: ethers.utils.parseUnits('1', 'gwei'),
+        maximumBaseFee: uint128Max,
+      },
```

Comment:
This feels like hidden config. Shouldn't these go in a `deploy-config/{{NETWORK}}.json` file?

Reply:
Your reasoning make sense to me. Putting it into a short comment here would be good.